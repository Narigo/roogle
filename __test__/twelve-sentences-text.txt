Here is a sentence that might be filtered out, since it is number one.
Here is a sentence that might be filtered out, since it is number two.
Here is a sentence that might be filtered out, since it is number three.
Here is a sentence that might be filtered out, since it is number four.
Here is a sentence that might be filtered out, since it is number five.
Here is a sentence that might be filtered out, since it is number six.
Here is a sentence that might be filtered out, since it is number seven.
Here is a sentence that might be filtered out, since it is number eight.
Here is a sentence that might be filtered out, since it is number nine.
Here is a sentence that might be filtered out, since it is number ten.
Here is a sentence that might be filtered out, since it is number eleven.
Here is a sentence that might be filtered out, since it is number twelve.
