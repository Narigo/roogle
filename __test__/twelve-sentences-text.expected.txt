open "https://www.google.com/search?q=Here%20is%20a%20sentence%20that%20might%20be%20filtered%20out%20since%20it%20is%20number%20one"
open "https://www.google.com/search?q=Here%20is%20a%20sentence%20that%20might%20be%20filtered%20out%20since%20it%20is%20number%20two"
open "https://www.google.com/search?q=Here%20is%20a%20sentence%20that%20might%20be%20filtered%20out%20since%20it%20is%20number%20three"
open "https://www.google.com/search?q=Here%20is%20a%20sentence%20that%20might%20be%20filtered%20out%20since%20it%20is%20number%20four"
open "https://www.google.com/search?q=Here%20is%20a%20sentence%20that%20might%20be%20filtered%20out%20since%20it%20is%20number%20five"
open "https://www.google.com/search?q=Here%20is%20a%20sentence%20that%20might%20be%20filtered%20out%20since%20it%20is%20number%20six"
open "https://www.google.com/search?q=Here%20is%20a%20sentence%20that%20might%20be%20filtered%20out%20since%20it%20is%20number%20seven"
open "https://www.google.com/search?q=Here%20is%20a%20sentence%20that%20might%20be%20filtered%20out%20since%20it%20is%20number%20eight"
open "https://www.google.com/search?q=Here%20is%20a%20sentence%20that%20might%20be%20filtered%20out%20since%20it%20is%20number%20nine"
open "https://www.google.com/search?q=Here%20is%20a%20sentence%20that%20might%20be%20filtered%20out%20since%20it%20is%20number%20ten"
open "https://www.google.com/search?q=Here%20is%20a%20sentence%20that%20might%20be%20filtered%20out%20since%20it%20is%20number%20eleven"
open "https://www.google.com/search?q=Here%20is%20a%20sentence%20that%20might%20be%20filtered%20out%20since%20it%20is%20number%20twelve"
